src/entries/popup/pages/home/Tokens.tsx

```diff
@@ -345,6 +348,19 @@ export const AssetRow = memo(function AssetRow({
   const priceChangeDisplay = priceChange?.length ? priceChange : '-';
   const priceChangeColor =
     priceChangeDisplay[0] !== '-' ? 'green' : 'labelTertiary';
+  const { shouldApproximate } = useMemo(
+    () =>
+      comparePlatformToCalculatedValue({
+        platformAmount: asset.platformValue?.amount,
+        calculatedAmount: asset.native.balance.amount,
+      }),
+    [asset.native.balance.amount, asset.platformValue?.amount],
+  );
+  const platformDisplay =
+    asset.platformValue?.display ?? asset.native.balance.display;
+  const displayWithApproximation = shouldApproximate
+    ? `~${platformDisplay}`
+    : platformDisplay;
```

Comment:
Confused a bit by the difference between values here and the need for the approximation display value. Is that a new Backend thing? I did see `~` values on quite a few assets and it wasn't obvious what exactly that meant

Reply:
Worth design review as well